packages/main/src/MultiComboBox.ts

```diff
@@ -445,10 +168,299 @@ const metadata = {
+	 * @type {boolean}
+	 * @name sap.ui.webc.main.MultiComboBox.prototype.disabled
+	 * @defaultvalue false
+	 * @public
+	 */
+	@property({ type: Boolean })
+	disabled!: boolean;
+
+	/**
+	 * Defines the value state of the component.
+	 * <br><br>
+	 * Available options are:
+	 * <ul>
+	 * <li><code>None</code></li>
+	 * <li><code>Error</code></li>
+	 * <li><code>Warning</code></li>
+	 * <li><code>Success</code></li>
+	 * <li><code>Information</code></li>
+	 * </ul>
+	 *
+	 * @type {sap.ui.webc.base.types.ValueState}
+	 * @name sap.ui.webc.main.MultiComboBox.prototype.valueState
+	 * @defaultvalue "None"
+	 * @public
+	 */
+	@property({ type: ValueState, defaultValue: ValueState.None })
+	valueState!: ValueState;
+
+	/**
+	 * Defines whether the component is read-only.
+	 * <br><br>
+	 * <b>Note:</b> A read-only component is not editable,
+	 * but still provides visual feedback upon user interaction.
+	 *
+	 * @type {boolean}
+	 * @name sap.ui.webc.main.MultiComboBox.prototype.readonly
+	 * @defaultvalue false
+	 * @public
+	 */
+	@property({ type: Boolean })
+	readonly!: boolean;
+
+	/**
+	 * Defines whether the component is required.
+	 *
+	 * @type {boolean}
+	 * @name sap.ui.webc.main.MultiComboBox.prototype.required
+	 * @defaultvalue false
+	 * @public
+	 * @since 1.0.0-rc.5
+	 */
+	@property({ type: Boolean })
+	required!: boolean;
+
+	/**
+	 * Defines the filter type of the component.
+	 * Available options are: <code>StartsWithPerTerm</code>, <code>StartsWith</code>, <code>Contains</code> and <code>None</code>.
+	 *
+	 * @type {string}
+	 * @name sap.ui.webc.main.MultiComboBox.prototype.filter
+	 * @defaultvalue "StartsWithPerTerm"
+	 * @public
+	 */
+	@property({ type: ComboBoxFilter, defaultValue: ComboBoxFilter.StartsWithPerTerm })
+	filter!: ComboBoxFilter;
+
+	/**
+	 * Indicates whether the dropdown is open. True if the dropdown is open, false otherwise.
+	 *
+	 * @type {boolean}
+	 * @name sap.ui.webc.main.MultiComboBox.prototype.filter
+	 * @defaultvalue false
+	 * @readonly
+	 * @since 1.0.0-rc.5
+	 * @public
+	 */
+	@property({ type: Boolean })
+	open!: boolean;
+
+	/**
+	 * Defines the accessible ARIA name of the component.
+	 *
+	 * @type {string}
+	 * @name sap.ui.webc.main.MultiComboBox.prototype.accessibleName
+	 * @defaultvalue: ""
+	 * @public
+	 * @since 1.4.0
+	 */
+	@property()
+	accessibleName!: string;
+
+	/**
+	 * Receives id(or many ids) of the elements that label the component.
+	 *
+	 * @type {string}
+	 * @name sap.ui.webc.main.MultiComboBox.prototype.accessibleNameRef
+	 * @defaultvalue ""
+	 * @public
+	 * @since 1.4.0
+	 */
+	@property({ defaultValue: "" })
+	accessibleNameRef!: string;
+
+	@property({ type: Object, noAttribute: true, multiple: true })
+	_filteredItems!: Array<IMultiComboBoxItem>;
+
+	@property({ type: Boolean })
+	filterSelected!: boolean;
+
+	@property({ type: Boolean })
+	focused!: boolean;
+
+	@property({ type: Boolean })
+	_tokenizerFocused!: boolean;
+
+	@property({ type: Boolean })
```

Comment:
previously this property had noAttribute: true. Why we don't need it anymore?

Reply:
returned it and also applied to other properties which do not need that